onnx_chainer/export.py

```diff
@@ -259,6 +266,16 @@ def export(model, args, filename=None, export_params=True,
             outputs.grad = numpy.ones_like(outputs.array)
             outputs.backward()
 
+    out_scoped_param_names = set(o.inputs.keys()) - param_names -\
+        network_input_names
+    if out_scoped_param_names:
+        for name in out_scoped_param_names:
+            param = o.inputs[name]
+            initializers.append(convert_parameter(param))
```

Comment:
JFYI: currently, ONNX-chainer is using input+initializer instead of `Constant` for this kind of constant values. However, I think `Constant` is more appropriate and input+initializer should be only for network parameters. This is OK, but it'd be great if we can eventually fix it.

Reply:
agree, I issued #82

Reply:
Thanks!